keyboards/neopad/readme.md

```diff
@@ -0,0 +1,25 @@
+# Neopad
+
+Neopad is a little macro pad with four switches and two rotary encoders, made by through hole components only.
+_Actually the number of switches is six, because even the encoder are allowed to click._
+
+![Neopad](https://i.imgur.com/TgOkj2F.jpg "Neopad first proto")
```

Comment:
```suggestion
![Neopad](https://i.imgur.com/TgOkj2Fh.jpg "Neopad first proto")
```

Reply:
Hi, I think my URL is correct. I don't see why I should change it LOL

Reply:
It is correct, the `h` suffix just makes it a bit smaller: https://api.imgur.com/models/image#thumbs

Reply:
I see, thank you for explain this to me.
Updated